### ISSUE TITLE
Fix the cause of frequent P2P disconnects

### ISSUE THREAD

**Issue by MaciejBaj** [LiskHQ/lisk#1906]
### Expected behavior
Peers shouldn't disconnect frequently if the network connection is kept.
### Actual behavior
Peers are disconnecting from each other frequently.
### Steps to reproduce
To investigate.
### Which version(s) does this affect? (Environment, OS, etc...)
1.0.0

**Comment by diego-G** [comment]
I've just had this error probably related to the same issue:
```
0|lisk     | [inf] 2018-04-23 12:57:12 | Broadhash consensus now 2.1 %
0|lisk     | [inf] 2018-04-23 12:57:12 | Finished sync
0|lisk     | Unable to find resolving function for procedure updateMyself with signature 1524488230875_8657
0|lisk     | [inf] 2018-04-23 12:57:14 | Removed peer - 88.130.49.146:5000
0|lisk     | [inf] 2018-04-23 12:57:14 | Removed peer - 213.86.144.42:8001
0|lisk     | [inf] 2018-04-23 12:57:14 | Removed peer - 213.86.144.42:5000
0|lisk     | [inf] 2018-04-23 12:57:14 | Removed peer - 159.89.52.163:5000
0|lisk     | [inf] 2018-04-23 12:57:15 | Releasing enqueued broadcasts
0|lisk     | [inf] 2018-04-23 12:57:15 | Broadhash consensus now 2.1 %
0|lisk     | [inf] 2018-04-23 12:57:15 | Broadcasts released: 1
0|lisk     | [inf] 2018-04-23 12:57:16 | Removed peer - 67.207.91.90:5000
0|lisk     | [inf] 2018-04-23 12:57:16 | Removed peer - 104.236.119.111:5000
0|lisk     | [inf] 2018-04-23 12:57:16 | Removed peer - 159.203.44.89:5000
0|lisk     | [inf] 2018-04-23 12:57:16 | Removed peer - 95.91.246.93:8001
0|lisk     | [inf] 2018-04-23 12:57:16 | Removed peer - 206.189.63.111:5000
0|lisk     | [inf] 2018-04-23 12:57:16 | Removed peer - 88.130.49.115:5000
0|lisk     | [inf] 2018-04-23 12:57:20 | Releasing enqueued broadcasts
0|lisk     | [inf] 2018-04-23 12:57:20 | Queue empty
0|lisk     | [inf] 2018-04-23 12:57:21 | Removed peer - 159.203.44.89:5000
0|lisk     | [inf] 2018-04-23 12:57:21 | Removed peer - 95.91.246.93:8001
0|lisk     | [inf] 2018-04-23 12:57:21 | Removed peer - 206.189.63.111:5000
0|lisk     | [inf] 2018-04-23 12:57:21 | Removed peer - 88.130.49.115:5000
0|lisk     | [inf] 2018-04-23 12:57:22 | Starting sync
0|lisk     | [inf] 2018-04-23 12:57:22 | Broadhash consensus now 2.1 %
0|lisk     | [inf] 2018-04-23 12:57:22 | Looking for common block with: 159.89.50.111:5000
0|lisk     | [inf] 2018-04-23 12:57:23 | Found common block: 5310860023742676621 with: 159.89.50.111:5000
0|lisk     | [inf] 2018-04-23 12:57:23 | Loading blocks from: 159.89.50.111:5000
0|lisk     | [ERR] 2018-04-23 12:57:23 | Expected generator: f827f60366fae9f9ed65384979de780f4a18c6dbfbefb1c7d100957dde51a06d Received generator: cf8a3bf23d1936a34facc4ff63d86d21cc2e1ac17e0010035dc3ef7ae85010dc
0|lisk     | [inf] 2018-04-23 12:57:23 | Fork - {"delegate":"cf8a3bf23d1936a34facc4ff63d86d21cc2e1ac17e0010035dc3ef7ae85010dc","block":{"id":"18383093375590430649","timestamp":60379040,"height":46260,"previousBlock":"5310860023742676621"},"cause":3}
0|lisk     | [ERR] 2018-04-23 12:57:23 | Error loading blocks: Failed to verify slot: 6037904
0|lisk     | [ERR] 2018-04-23 12:57:23 | Failed to load blocks from: 159.89.50.111:5000
0|lisk     | [inf] 2018-04-23 12:57:23 | Broadhash consensus now 2.1 %
0|lisk     | [inf] 2018-04-23 12:57:23 | Finished sync
````
Node bumps into error `nable to find resolving function for procedure updateMyself with signature 1524488230875_8657` and then it starts to fork forever.

**Comment by MaciejBaj** [comment]
Closed by #1932.
